src/module-loader.ts

```diff
@@ -13,7 +13,7 @@ const getPackageType = require('get-package-type')
  * Defines file extension resolution when source files do not have an extension.
  */
 // eslint-disable-next-line camelcase
-const s_EXTENSIONS: string[] = ['.js', '.mjs', '.cjs']
+const s_EXTENSIONS: string[] = ['.ts', '.js', '.mjs', '.cjs']
```

Comment:
Should `.jsx` or `.tsx` file extensions be included?

Reply:
Yes. Anything in oclif that prevents one from using react?

Reply:
@peternhale I just don't see the use case for using jsx syntax in a CLI. I'd rather not support it until someone comes along with a convincing use case

Reply:
I definitely want to play around with https://github.com/vadimdemedes/ink at some point. Might add it then 😄